symplyphysics/core/geometry/line.py

```diff
@@ -0,0 +1,20 @@
+from sympy import symbols, Point, Line, Expr, solve
+
+def two_point_function(p1: Point, p2: Point, x: Expr) -> Expr:
```

Comment:
Can we narrow down `x` type to `Symbol`?

Reply:
That would be too narrow a type. We might want to have, say, `f(x)` as the `x`, and the closest type it shares with `Symbol` in the MRO hierarchy is `Expr`.

Reply:
I doubt we'll ever want `f(x)` as parameter here.

Reply:
Okay, we can have `Symbol` for now